bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/handler/OpenSprinklerStationHandler.java

```diff
@@ -141,7 +157,7 @@ protected State getStationState(int stationId) {
      * @return State representation for the channel.
      */
     @Nullable
-    protected State getRemainingWaterTime(int stationId) {
+    private State getRemainingWaterTime(int stationId) {
```

Comment:
```suggestion
    private @Nullable State getRemainingWaterTime(int stationId) {
```

see https://my.openhab.org/docs/developer/guidelines.html#null-annotations